src/modules/functions.js

```diff
@@ -0,0 +1,112 @@
+import removeIcon from '../img/remove.png';
+
+const icon3 = new Image();
+icon3.src = removeIcon;
+const todoList = [];
+
+export default class Todo {
+  constructor(todo) {
+    this.index = todoList.length + 1;
+    this.description = todo;
+    this.id = Date.now().toString();
+    this.completed = false;
+  }
+
+    static display = () => {
+      const todoItems = document.getElementById('id-list');
+      const todoList = JSON.parse(localStorage.getItem('todoList') || '[]');
+      let inputCheck = '';
+      todoItems.innerHTML = '';
+      todoList.forEach((task) => {
+        if (task.completed === false) {
+          inputCheck = '';
+        } else {
+          inputCheck = 'checked';
+        }
+        todoItems.innerHTML += `
+          <li>
+            <span class="item-info">
+              <input ${inputCheck} type="checkbox" class="check" id="input${task.index}">
+              <input id="${task.index}" class="task-item" value="${task.description}">
+            </span>
+            <span id="item-icon">
+              <img src="${icon3.src}" class="remove" id="${task.id}" alt="Remove Task" title="Remove Task">
+            </span>
+          </li>
+          `;
+      });
+    }
+/* eslint-disable */
+    static add = () => {
+      const alert = document.querySelector('.notice');
+      let arr; let
+        todoList = [];
+      const inputForm = document.querySelector('#form-data');
+      const doInput = document.querySelector('#id-input');
+      inputForm.addEventListener('submit', (event) => {
+        event.preventDefault();
+        todoList = JSON.parse(localStorage.getItem('todoList') || arr);
+        if (doInput.value !== '') {
+          const todoItem = new Todo(doInput.value);
+          todoList.push(todoItem);
+          todoList.forEach((todo) => {
+            todo.index;
+          });
+          localStorage.setItem('todoList', JSON.stringify(todoList));
+          doInput.value = '';
+          window.location.reload();
+        } else {
+          alert.style.display = 'flex';
+          alert.innerText = 'Invalid input';
+        }
+      });
+    }
```

Comment:
- [ ] when i try adding an item to the list the following error pops up 
```Uncaught SyntaxError: "undefined" is not valid JSON
    at JSON.parse (<anonymous>)
    at HTMLFormElement.eval (functions.js:53:25)
```
It will be great to make sure the app adds an item to the list and also it can filter the list, mark items as completed, and update the items in the local storage.

Due to the error above i was not able to test those functionalities. Kindly fix and submit your project for another review.